.github/workflows/test.yml

```diff
@@ -62,6 +56,13 @@ jobs:
       - name: Setup Gradle options
         run: echo "GRADLE_OPTIONS=-Porg.gradle.java.installations.auto-detect=false -Porg.gradle.java.installations.fromEnv=JAVA_HOME_11_x64,JAVA_HOME_8_x64" >> $GITHUB_ENV
 
+      - name: Create Javadocs
+        run: |
+          cd newrelic-java-agent/newrelic-api 
+          ../gradlew javadoc -Prelease=true -PagentVersion=8.0.0
```

Comment:
This is the cmd that was failing because it was missing the jdk8 property.
One of those other scripts did add that property, so it had to be run before this gradle cmd.